include/khiva/dimensionality.h

```diff
@@ -187,10 +193,11 @@ KHIVAAPI af::array SAX(af::array a, int alphabetSize);
  *
  * @param pointList Expects an input vector of points.
  * @param numPoints Sets the number of points returned after the execution of the method.
+ * * @param scale Sets the precision used to compute the areas of the triangularization, the longer, the more accurate.
  *
  * @return std:vector<khiva::dimensionality::Point> where the number of points has been reduced to numPoints.
  */
-KHIVAAPI std::vector<Point> visvalingam(std::vector<Point> pointList, int numPoints);
+KHIVAAPI std::vector<khiva::dimensionality::Point> visvalingam(std::vector<khiva::dimensionality::Point> pointList, int64_t numPoints, int64_t scale = 10000000);
```

Comment:
Just a comment of something that can be done in C++ and you might not know. You can use `int64_t scale = 1e7` or `int64_t scale = 10'000'000`

Reply:
I knew the former but not the later. xD